Yona/Yona/DaySingleActivityDetail.swift

```diff
@@ -26,12 +26,20 @@ class DaySingleActivityDetail: NSObject {
     var goalName : String = ""
     var goalType: String?
     
-    var totalActivityDurationMinutes : Int
-    var goalAccomplished : Bool
-    var totalMinutesBeyondGoal : Int
+    var totalActivityDurationMinutes : Int = 0
+    var goalAccomplished : Bool = false
+    var totalMinutesBeyondGoal : Int = 0
     var maxDurationMinutes: Int = 0
     
     init(data : BodyDataDictionary, allGoals : [Goal]) {
+        super.init()
+        retrieveDayofWeekFromDate(data)
+        retrieveActivityDetail(data)
+        retrieveUserLinks(data)
+        retrieveGoalDetails(allGoals)
+    }
+
+    fileprivate func retrieveDayofWeekFromDate(_ data: BodyDataDictionary) {
```

Comment:
Correct case: ``retrieveDayOfWeekFromDate`` (every word with a capital, so ``Of``)

Reply:
Done.